report_pre_thesis/src/index.tex

```diff
@@ -70,9 +72,26 @@ \subsection{Problem Statement Proposal}
 % This raises a critical question: How can the MOC model be refined to enhance its accuracy and robustness, ensuring more reliable predictions that can better support the objectives of the MSL mission?
 % \subsection{comment-end}
 
-In this work, we aim to identify potential limitations and areas of improvement in the Multivariate Oxide Composition model's predictive accuracy and robustness in predicting the composition of major oxides in Martian geological samples using Earth-based calibration data.
+\section{Definition}
+Given a set \(E\) of eight major oxides detectable using LIBS, \(E = \{SiO_2, TiO_2, Al_2 O_3, FeOT, MgO, CaO, Na_2O, K_2O\}\), and considering \(\mathcal{P}(E)\) as the power set of \(E\) which represents all possible combinations of these oxides, we define \(K \subseteq \mathcal{P}(E) \) as the set of oxide compositions detected by LIBS.
+
+Let $S$ be the set of all possible spectra, where each spectrum $\mathbf{s} \in S$ is a multidimensional vector in $\mathbb{R}^n$ with $n$ being the number of wavelengths captured by the LIBS spectrometer for a single laser pulse. $\mathbf{s}$ holds the intensities of light emitted from the plasma that is created from the laser ablation process.
```

Comment:
Actually, we can't do this because we need to distinguish wavelengths! The value of the wavelength matters greatly in the predictions, because you need to consider the wavelength where you see a high/low intensity